ui/StatusQ/src/StatusQ/Core/Utils/OperatorsUtils.qml

```diff
@@ -30,7 +30,9 @@ QtObject {
             case OperatorsUtils.Operators.Or:
                 return root.orOperatorText
             case OperatorsUtils.Operators.None:
+            default:
                 return ""
+
```

Comment:
Is this whole component still needed?

Reply:
I would suggest keeping it because I kept the logic of the operators in `StatusItemSelector` as it is a general component it might be useful to have this in the future.

Reply:
Cool